Updated D4

HadesGeneral opened the pull request:
Added a better description of how the app works and cleaned up a bit of the document.

HadesGeneral committed:
Updated D4

Added a better description of how the app works and cleaned up a bit of the document.

HadesGeneral committed:
Created Week 10 Reflection

Made and completed Reflection

HadesGeneral committed:
Updated File

Added two more objectives

HadesGeneral committed:
Update File

Added in Testing section

HadesGeneral committed:
Updated File

HadesGeneral committed:
created week 11 reflection.

Created file

HadesGeneral committed:
Updated File

Did spell check

HadesGeneral committed:
Created Research folder, and YouTube links

Created a folder for overall research and a file for youtube links

belainehi commented on Reflections/Week_11_Reflection.md:
```diff
@@ -0,0 +1,26 @@
+  - Made the decision to switch from a linear acuator since the part I need is coming from Denmark and I can't rely in it coming in time. I have deciced to move into using a converbelt as the weight shifting part.
+  - created a gear system that shows off the speed difference between using only the motor, using one set of gears, and two sets of gears. With each incrimenting in speed respeciivly. 
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+  - I faced difficulty figuring out an alterative to the linear actuator since it was the original idea for the weight shifting part. A converbelt is the next thing, which could be difficult since the belt needs to be tight enouch to not buckle under the wieght. 
```
This could be a drawing of the rail you are proposing.

belainehi commented on Reflections/Week_11_Reflection.md:
```diff
@@ -0,0 +1,26 @@
+    - Specific: Try to find lego conver belts on youtube or otherplaces to get ideas on how to create one that's best for my system.
+    - Measurable: 1-3 links should be what I aim for.
+    - Achiveable: This can be done within the next report out. 
+  - See if i can renforce the wheel system.
```
Have you considered adding a drawing of the wheel you are planing on building?

belainehi commented on Reflections/Week_11_Reflection.md:
```diff
@@ -0,0 +1,26 @@
+    - Specific: The part where the wheels connect to the rest of the system is not stable enough to make sure it can't move side to side. it only needs to be able to move up or down to simulate suspention, similar to car suspension.
+    - Achiveable: This is something i have been doing slowly since i first built the system. With the extra parts it is more achievable.
+    - Measureble: If the wheels become more restriced when it comes to moving the wheels side to side then I know i have done it.
+4. What resources will you use to accomplish your goals for the upcoming week? (please list out the resources)
```
Can you be more specific about what type of parts you plan to use? Think of it as if someone wanted to recreate your project what items would they need to buy?

belainehi replied:
What type of gears, wheels...?

belainehi commented on Reflections/Week_11_Reflection.md:
```diff
@@ -0,0 +1,26 @@
+    - wheels
+    - basic blocks
+    - axles
+  - ducktape
```
Duct tape

belainehi commented on Reflections/Week_11_Reflection.md:
```diff
@@ -0,0 +1,26 @@
+  - Made the decision to switch from a linear acuator since the part I need is coming from Denmark and I can't rely in it coming in time. I have deciced to move into using a converbelt as the weight shifting part.
+  - created a gear system that shows off the speed difference between using only the motor, using one set of gears, and two sets of gears. With each incrimenting in speed respeciivly. 
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+  - I faced difficulty figuring out an alterative to the linear actuator since it was the original idea for the weight shifting part. A converbelt is the next thing, which could be difficult since the belt needs to be tight enouch to not buckle under the wieght. 
```
Congrats on getting the linear actuator!

belainehi commented on Reflections/Week_11_Reflection.md:
```diff
@@ -0,0 +1,26 @@
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+  - Got the old system working on the lonboard. But, this only worked moving forward. The ystem can't suport more than 2.5 lb and the longboard needs more than that to see a difference in its turing capability.
+  - Made the decision to switch from a linear acuator since the part I need is coming from Denmark and I can't rely in it coming in time. I have deciced to move into using a converbelt as the weight shifting part.
+  - created a gear system that shows off the speed difference between using only the motor, using one set of gears, and two sets of gears. With each incrimenting in speed respeciivly. 
```
This sounds great! It would be easier to understand with more visualization.

HadesGeneral committed:
Updated File

Added linear actuator link

HadesGeneral committed:
Added Files

added photos showing off the linear actuator prototype

HadesGeneral committed:
Added Photots

Added wheel support photos

HadesGeneral committed:
Updated File

added the goals for week 11 reflection, changed the conveyor belt goal to the linear actuator since I got the part in.

belainehi commented on Reflections/Week_10_Reflection.md:
```diff
@@ -0,0 +1,14 @@
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+  - Got the most basic linear actuator made to get an understanding of what I need. 
+  - Updated some of the files throughout the project
+  - Have a longboard to test if the system currently has the power to move it as well.
```
Testing your project on two different sized boards seems like a great idea for seeing how it performs with the two different weights of the boards.

belainehi commented on Reflections/Week_10_Reflection.md:
```diff
@@ -0,0 +1,14 @@
+  - See if the current system has the power to move the longboard. This could take about 2 hours to complete.
+    - Specific: Take the built system off the old board, and stransfer it too the new longboard. Run it and see if it struggles to move the longbaord or moves it without trouble.
+    - Measurable: Time the smaller board at how mlong it takes to get to the end of the length I have specified. Using the time it takes to determine if it struggles or not.
+    - Achievable: This can be done easily, but the difficulty might come from the length of the board difference, otherwise this is achievable.
```
Will you be able to use the same linear actuator for different sized skateboards? Will you use two different lead screws?

belainehi commented on Reflections/Week_10_Reflection.md:
```diff
@@ -0,0 +1,14 @@
+    - Specific: Take the built system off the old board, and stransfer it too the new longboard. Run it and see if it struggles to move the longbaord or moves it without trouble.
+    - Measurable: Time the smaller board at how mlong it takes to get to the end of the length I have specified. Using the time it takes to determine if it struggles or not.
+    - Achievable: This can be done easily, but the difficulty might come from the length of the board difference, otherwise this is achievable.
+7. What resources will you use to accomplish your goals for the upcoming week? (please list out the resources)
```
Can you write out what the type of long board is and the components of the current built system?
